DXMainClient/DXGUI/Multiplayer/MapTextureCacheManager.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+using DTAClient.Domain.Multiplayer;
+
+using SixLabors.ImageSharp;
+
+namespace DTAClient.DXGUI.Multiplayer
+{
+    public class MapTextureCacheManager : IDisposable
+    {
+        public const int MaxCacheSize = 100;
+        public const int SleepIntervalMS = 100;
+
+        private readonly ConcurrentDictionary<Map, Image> mapTextures = [];
+
+        private readonly ConcurrentDictionary<Map, byte> missedMaps = [];
```

Comment:
1. Why is this a dictionary when this should be a set? If the reason is that there's no concurrent set it's better to use some empty type like `System.Reactive.Unit`, for instance, as it better conveys the purpose.
2. IMO this should be brought higher and renamed to `pendingMaps`, it's not `missing`, the maps are just waiting to be processed.

Reply:
So what do you suggest? I think we can't introduce RX here right now just because of introducing the unit type

Reply:
Well, I don't see a big problem with introducing Rx, but I guess you could make an empty type or just copy the unit type from `System.Reactive`, and later we could just replace it when we introduce Rx.